src/devices_models/devices/electric_loads.jl

```diff
@@ -33,6 +33,23 @@ get_multiplier_value(::TimeSeriesParameter, d::PSY.ElectricLoad, ::AbstractContr
 
 #! format: on
 
+function initialize_timeseries_names(
```

Comment:
@daniel-thom what do you think about these names to set-up the default values of the time series and attributes of the models. Should we re-name this methods to something more clear?

Reply:
I would probably call it `get_time_series_name_definitions`.

Note that we use `time_series` (two words).

Reply:
How about `get_time_series_default_names`